packages/sp/files/types.ts

```diff
@@ -46,7 +46,7 @@ export class _Files extends _SharePointQueryableCollection<IFileInfo[]> {
      * @returns The new File and the raw response.
      */
     @tag("fis.add")
-    public async add(url: string, content: string | ArrayBuffer | Blob, shouldOverWrite = true): Promise<IFileAddResult> {
+    public async add(url: string, content: string | ArrayBuffer | Blob | any, shouldOverWrite = true): Promise<IFileAddResult> {
```

Comment:
`any` is what fetch defines as its input anyway

Reply:
@patrick-rodgers 
> For file add we can just change the param type to any, no reason to list types and append any as it no longer is restrictive IMO

We can collapse this to `any` or add some types representing node streams (would have to investigate if there's a single root type for all streams) but this value is passed to fetch, which accepts `any`.

Keeping the other types has one advantage - IDE suggestions. "some type hints or any" is better than just "any". 

I'd keep the list as long as possible for the sake of good IDE tips.